docs/eloquent-models/schema-builder.txt

```diff
@@ -0,0 +1,16 @@
+.. _laravel-eloquent-model-schema-builder:
+
+==============
+Schema Builder
+==============
+
+This page will show how to use Laravel Schemas to manage indexes and show
```

Comment:
This will be totally new contents, right?

Reply:
I think so -- the Create example might be similar to the existing content, but the rest of the examples need to be created from scratch.